test/case/ietf_system/hostname.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python3
 
-import random, string
-
+import random
+import string
+import re
 import infamy
 
 with infamy.Test() as test:
     with test.step("Initialize"):
         env = infamy.Env(infamy.std_topology("1x1"))
         target = env.attach("target", "mgmt")
+        tgt = env.attach("target", "mgmt", "ssh")
```

Comment:
I think we should settle on a convention for these names. The user test used `tgtssh` for the SSH channel. Maybe there's something even better, but I think that `target` and `tgt` may be confusing to new readers.

Reply:
OK, fixing.